From: Jason Rhinelander <jason@imaginary.ca>
Date: Mon, 16 Dec 2019 23:38:52 -0400
Subject: Bump PATCH to 9 for pre-release version

---
 httpserver/version.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/httpserver/version.h b/httpserver/version.h
index a34a505..a0f77f6 100644
--- a/httpserver/version.h
+++ b/httpserver/version.h
@@ -6,7 +6,7 @@
 
 #define VERSION_MAJOR 1
 #define VERSION_MINOR 0
-#define VERSION_PATCH 8
+#define VERSION_PATCH 9
 
 #define LOKI_STRINGIFY2(val) #val
 #define LOKI_STRINGIFY(val) LOKI_STRINGIFY2(val)
