From: Jason Rhinelander <jason@imaginary.ca>
Date: Tue, 6 Jul 2021 21:28:23 -0300
Subject: Old cmake sqlite include dir workaround

---
 external/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/external/CMakeLists.txt b/external/CMakeLists.txt
index d5d7238..0505d66 100644
--- a/external/CMakeLists.txt
+++ b/external/CMakeLists.txt
@@ -207,7 +207,7 @@ if(NOT BUILD_STATIC_DEPS) # Under BUILD_STATIC_DEPS the SQLite::SQLite3 target i
     add_library(SQLite::SQLite3 STATIC IMPORTED GLOBAL)
     add_dependencies(SQLite::SQLite3 sqlite3_external)
     set_target_properties(SQLite::SQLite3 PROPERTIES IMPORTED_LOCATION ${CMAKE_CURRENT_BINARY_DIR}/sqlite3/lib/libsqlite3.a)
-    target_include_directories(SQLite::SQLite3 INTERFACE ${CMAKE_CURRENT_BINARY_DIR}/sqlite3/include)
+    set_target_properties(SQLite::SQLite3 PROPERTIES INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_CURRENT_BINARY_DIR}/sqlite3/include)
   endif()
 endif()
 
