From: Jason Rhinelander <jason@imaginary.ca>
Date: Sat, 21 Dec 2019 04:26:16 -0400
Subject: local boost update

---
 utils/CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/utils/CMakeLists.txt b/utils/CMakeLists.txt
index d7f39b7..9e29d7f 100644
--- a/utils/CMakeLists.txt
+++ b/utils/CMakeLists.txt
@@ -6,11 +6,6 @@ add_library(utils STATIC
     src/utils.cpp
 )
 
-find_package(Boost
-    REQUIRED
-    filesystem
-)
-
 set_property(TARGET utils PROPERTY CXX_STANDARD 14)
 
 target_include_directories(utils
