From: Jason Rhinelander <jason@imaginary.ca>
Date: Mon, 21 Aug 2023 13:01:17 -0300
Subject: Don't count single-message boxes as accounts

---
 oxenss/snode/service_node.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/oxenss/snode/service_node.cpp b/oxenss/snode/service_node.cpp
index b6b6ce6..6e303c2 100644
--- a/oxenss/snode/service_node.cpp
+++ b/oxenss/snode/service_node.cpp
@@ -1252,6 +1252,10 @@ std::string ServiceNode::get_stats() const {
     std::vector<int> counts = db_->get_message_counts();
     int64_t total = std::accumulate(counts.begin(), counts.end(), int64_t{0});
 
+    counts.erase(
+            std::remove_if(counts.begin(), counts.end(), [](int c) { return c < 2; }),
+            counts.end());
+
     // If less than 5 our iterators below could end up at the same position, so just require at
     // least 5 rather than worrying about that case:
     if (counts.size() >= 5) {
