From: Jason Rhinelander <jason@imaginary.ca>
Date: Wed, 26 Feb 2020 19:20:05 -0400
Subject: Fix fmt linking

---
 common/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/CMakeLists.txt b/common/CMakeLists.txt
index 8958fd9..4aee279 100644
--- a/common/CMakeLists.txt
+++ b/common/CMakeLists.txt
@@ -24,6 +24,7 @@ set_property(TARGET common PROPERTY CXX_STANDARD 14)
 find_package(spdlog)
 
 target_link_libraries(common PUBLIC spdlog::spdlog ${Boost_LIBRARIES})
+target_compile_definitions(common PUBLIC FMT_HEADER_ONLY)
 target_include_directories(common PUBLIC
     ${CMAKE_CURRENT_LIST_DIR}/include
     ${Boost_INCLUDE_DIRS}
