From: Jason Rhinelander <jason@imaginary.ca>
Date: Mon, 18 Jan 2021 17:56:03 -0400
Subject: Old cmake compat fix for oxenmq

---
 CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dc29efe..00f60b6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,8 +76,10 @@ include(cmake/check_for_std_filesystem.cmake)
 if(NOT BUILD_STATIC_DEPS)
 endif()
 
-pkg_check_modules(OXENMQ REQUIRED IMPORTED_TARGET GLOBAL liboxenmq>=1.2.3)
-add_library(oxenmq::oxenmq ALIAS PkgConfig::OXENMQ)
+pkg_check_modules(OXENMQ REQUIRED IMPORTED_TARGET liboxenmq>=1.2.3)
+add_library(oxenmq_pkgconfig INTERFACE)
+target_link_libraries(oxenmq_pkgconfig INTERFACE PkgConfig::OXENMQ)
+add_library(oxenmq::oxenmq ALIAS oxenmq_pkgconfig)
 
 add_subdirectory(common)
 add_subdirectory(utils)
