From: Jason Rhinelander <jason@imaginary.ca>
Date: Wed, 8 Apr 2020 13:16:01 -0300
Subject: sodium fix

---
 crypto/src/lokid_key.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/crypto/src/lokid_key.cpp b/crypto/src/lokid_key.cpp
index cf063d9..8d79512 100644
--- a/crypto/src/lokid_key.cpp
+++ b/crypto/src/lokid_key.cpp
@@ -47,10 +47,10 @@ private_key_ed25519_t::from_hex(const std::string& sc_hex) {
 }
 
 public_key_t derive_pubkey_legacy(const private_key_t& private_key) {
-    ge25519_p3 A;
-    ge25519_scalarmult_base(&A, private_key.data());
+    ge_p3 A;
+    ge_scalarmult_base(&A, private_key.data());
     public_key_t publicKey;
-    ge25519_p3_tobytes(publicKey.data(), &A);
+    ge_p3_tobytes(publicKey.data(), &A);
 
     return publicKey;
 }
