From: Jason Rhinelander <jason@imaginary.ca>
Date: Sat, 16 Jan 2021 01:11:50 -0400
Subject: oxenmq target fix

---
 CMakeLists.txt            | 6 ++++--
 httpserver/CMakeLists.txt | 3 +--
 2 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 422384d..7a2e434 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -73,8 +73,10 @@ link_libatomic()
 
 find_package(Boost REQUIRED)
 
-pkg_check_modules(LOKIMQ REQUIRED IMPORTED_TARGET liblokimq>=1.2)
-add_library(lokimq::lokimq ALIAS PkgConfig::LOKIMQ)
+pkg_check_modules(OXENMQ REQUIRED IMPORTED_TARGET liboxenmq>=1.2.3)
+add_library(oxenmq INTERFACE)
+target_link_libraries(oxenmq INTERFACE PkgConfig::OXENMQ)
+add_library(oxenmq::oxenmq ALIAS oxenmq)
 
 add_subdirectory(common)
 add_subdirectory(utils)
diff --git a/httpserver/CMakeLists.txt b/httpserver/CMakeLists.txt
index 0362c57..b34e35c 100644
--- a/httpserver/CMakeLists.txt
+++ b/httpserver/CMakeLists.txt
@@ -56,8 +56,7 @@ target_include_directories(httpserver_lib PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}")
 
 find_package(PkgConfig QUIET)
 
-pkg_check_modules(LOKIMQ REQUIRED IMPORTED_TARGET liblokimq>=1.2.0)
-target_link_libraries(httpserver PUBLIC PkgConfig::LOKIMQ)
+target_link_libraries(httpserver PUBLIC oxenmq::oxenmq)
 
 if(PKG_CONFIG_FOUND)
     pkg_check_modules(SYSTEMD libsystemd)
