From: Jason Rhinelander <jason@imaginary.ca>
Date: Sat, 16 Jan 2021 17:20:27 -0400
Subject: lokid-rpc-ip option fix

---
 httpserver/command_line.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/httpserver/command_line.cpp b/httpserver/command_line.cpp
index 3340b52..47dc04b 100644
--- a/httpserver/command_line.cpp
+++ b/httpserver/command_line.cpp
@@ -21,7 +21,7 @@ void command_line_parser::parse_args(int argc, char* argv[]) {
         ("data-dir", po::value(&options_.data_dir), "Path to persistent data (defaults to ~/.oxen/storage)")
         ("config-file", po::value(&config_file), "Path to custom config file (defaults to `storage-server.conf' inside --data-dir)")
         ("log-level", po::value(&options_.log_level), "Log verbosity level, see Log Levels below for accepted values")
-        ("oxend-rpc-ip", po::value(&options_.oxend_rpc_port), "RPC IP on which the local Oxen daemon is listening (usually localhost)")
+        ("oxend-rpc-ip", po::value(&options_.oxend_rpc_ip), "RPC IP on which the local Oxen daemon is listening (usually localhost)")
         ("oxend-rpc-port", po::value(&options_.oxend_rpc_port), "RPC port on which the local Oxen daemon is listening")
         ("lmq-port", po::value(&options_.lmq_port), "Port used by OxenMQ")
         ("testnet", po::bool_switch(&options_.testnet), "Start storage server in testnet mode")
@@ -37,6 +37,8 @@ void command_line_parser::parse_args(int argc, char* argv[]) {
         ("ip", po::value<std::string>(), "(unused)")
         ("port", po::value(&options_.port), "Port to listen on")
         ("oxend-key", po::value(&options_.oxend_key), "Legacy secret key (test only)")
+        ("lokid-rpc-ip", po::value(&options_.oxend_rpc_ip), "Backwards compatible option for oxend RPC IP")
+        ("lokid-rpc-port", po::value(&options_.oxend_rpc_port), "Backwards compatible option for oxend RPC port")
         ("oxend-x25519-key", po::value(&options_.oxend_x25519_key), "x25519 secret key (test only)")
         ("oxend-ed25519-key", po::value(&options_.oxend_ed25519_key), "ed25519 public key (test only)");
     // clang-format on
